src/core/lib/pick-layers.js

```diff
@@ -150,13 +150,17 @@ export function pickVisibleObjects(
   const uniqueInfos = new Map();
 
   pickInfos.forEach(pickInfo => {
-    const viewport = getViewportFromCoordinates({viewports}); // TODO - add coords
-    let info = createInfo([pickInfo.x / pixelRatio, pickInfo.y / pixelRatio], viewport);
```

Comment:
What happens to the `createInfo` function? Should it be deleted now? Or is it used somewhere else?

Reply:
It is still used in during `pickObjets` , only thing that is done there is to unproject the (x,y) to LatLng, if preferred I can remove that function and inline the code.

Reply:
Yes I think we should align the code paths.